solr_conf/conf/solrconfig.xml

```diff
@@ -85,7 +86,6 @@
 
         collection_title_tesim^5
 
-        objectId_tesim^5
```

Comment:
there is no point in boosting any sort of phrase or n-gram matches in a single token string.